### DISCUSSION TITLE
Why \AppData\Local\CEF is always created?

### DISCUSSION THREAD

**Discussion by zxcvqwerasdf** [cefsharp/CefSharp#3745] [General]
```
            string pathGlobal = Path.Combine(Application.StartupPath, "browser_instances", "GLOBAL");
            if (!Directory.Exists(pathGlobal))
            {
                Directory.CreateDirectory(pathGlobal);
            }
            CefSettings cefGlobal = new CefSettings();
            cefGlobal.RootCachePath = pathGlobal;
            cefGlobal.CachePath = pathGlobal;
            Cef.Initialize(cefGlobal);
            var browser = new ChromiumWebBrowser();
```
Why after this \AppData\Local\CEF still created? How to prevent it? Thx you.
Using CefSharp.OffScreen 91.1.230

**Comment by amaitland** [comment]
In future please use discussions section to ask questions. https://github.com/cefsharp/CefSharp/discussions

It is used for downloading of dictionaries and storage of other user specific content. 

You can customize the path by setting http://cefsharp.github.io/api/91.1.x/html/P_CefSharp_CefSettingsBase_UserDataPath.htm

I'm not aware of any option for disabling creation. See also https://bitbucket.org/chromiumembedded/cef/issues/3094/cef-force-creates-a-donwloads-directory-in

CEF has it's own support forum at https://magpcss.org/ceforum/index.php

**Reply by zxcvqwerasdf** [reply]
Thank you, it's working
